src/bundle-configuration/zookeeper-prod/runtime-config.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+export ZK_DATA_DIR=/var/lib/zookeeper
```

Comment:
/var/lib is typically where data is written by these things - check the ZK doco though. Where it writes by convention is where we should.